### ISSUE TITLE
CLI Commands and Project Source Code Compilation

### ISSUE THREAD

**Issue by Starnesc5780** [kumar488/461-team-repo#43]
CLI commands cannot be used at this point because they require the compilation of the project source code, and the source code requires the installation of the dependencies that are supposed to be given by the CLI command "run install". To test the rest of the code, the user will have to manually install all dependencies (listed on line 22 of src/cli.ts), then compile the project ("npx tsc"), and then the CLI will function as expected. Possible fix is to change the run script to install dependencies without the use of a TypeScript file, then have the script automatically compile the project source code.

**Comment by Parth1811** [comment]
Good catch! Please fix this and raise a PR. Ask for approval from TA on that before merging
